ui/app/styles/components/toolbar.scss

```diff
@@ -8,6 +8,10 @@ $spacing: 1.5em;
   margin-left: -$spacing / 4;
   margin-right: -$spacing / 4;
 
+  &.collapse + .toolbar {
+    margin-top: -$spacing / 2;
+  }
+
```

Comment:
I placed the toggle in a second `toolbar` and use this to bring them closer together. Maybe there’s a better approach? 🤔

Reply:
This is sorta in the same category as the above comment. No better approach currently. I like that this is targeted specifically at two toolbars with the collapse class though.